packages/teams-js/src/public/pages.ts

```diff
@@ -486,12 +529,17 @@ export namespace pages {
 
     /**
      * Checks if pages.backStack capability is supported currently
+     * @returns true if the pages.backStack capability is enabled in runtime.supports.pages.backStack and
+     * false if it is disabled
      */
     export function isSupported(): boolean {
       return runtime.supports.pages ? (runtime.supports.pages.backStack ? true : false) : false;
     }
   }
 
+  /**
+   * Namespace to interact with the full-trust part of the SDK
```

Comment:
Could we make this description more specific (for example, brief summary of full trust)?